<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Note</class>
 <widget class="QMainWindow" name="Note">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Note</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>594</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_All"/>
    <addaction name="separator"/>
    <addaction name="actionUndo"/>
    <addaction name="actionREDO"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <widget class="QMenu" name="menuAlingnment">
     <property name="title">
      <string>&amp;Alingnment</string>
     </property>
     <addaction name="actionLeft"/>
     <addaction name="actionCenter"/>
     <addaction name="actionRight"/>
     <addaction name="separator"/>
     <addaction name="actionJustify"/>
    </widget>
    <widget class="QMenu" name="menuFormatting">
     <property name="title">
      <string>&amp;Formatting</string>
     </property>
     <addaction name="actionBold"/>
     <addaction name="actionItalics"/>
     <addaction name="actionUnderline"/>
    </widget>
    <addaction name="actionFind"/>
    <addaction name="menuAlingnment"/>
    <addaction name="menuFormatting"/>
    <addaction name="actionWarp_Text"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoom_Out"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="separator"/>
    <addaction name="actionCheck_for_Updates"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menu_Tools"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_As"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionBold"/>
   <addaction name="actionItalics"/>
   <addaction name="actionUnderline"/>
   <addaction name="separator"/>
   <addaction name="actionExport"/>
   <addaction name="separator"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoom_Out"/>
   <addaction name="separator"/>
   <addaction name="actionLeft"/>
   <addaction name="actionCenter"/>
   <addaction name="actionRight"/>
   <addaction name="actionJustify"/>
   <addaction name="separator"/>
   <addaction name="actionPrint"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionCheck_for_Updates">
   <property name="text">
    <string>Check for &amp;Updates</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+U</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="Note_Res.qrc">
     <normaloff>:/new/icons/Icons/editcut.png</normaloff>:/new/icons/Icons/editcut.png</iconset>
   </property>
   <property name="text">
    <string>C&amp;ut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="Note_Res.qrc">
     <normaloff>:/new/icons/Icons/editcopy.png</normaloff>:/new/icons/Icons/editcopy.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="Note_Res.qrc">
     <normaloff>:/new/icons/Icons/editpaste.png</normaloff>:/new/icons/Icons/editpaste.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="text">
    <string>Select &amp;All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="Note_Res.qrc">
     <normaloff>:/new/icons/Icons/editundo.png</normaloff>:/new/icons/Icons/editundo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Undo </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionREDO">
   <property name="icon">
    <iconset resource="Note_Res.qrc">
     <normaloff>:/new/icons/Icons/editredo.png</normaloff>:/new/icons/Icons/editredo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>&amp;Find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionWarp_Text">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Warp Text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="Note_Res.qrc">
     <normaloff>:/new/icons/Icons/fileprint.png</normaloff>:/new/icons/Icons/fileprint.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Print</string>
   </property>
  </action>
  <action name="actionLeft">
   <property name="icon">
    <iconset resource="Note_Res.qrc">
     <normaloff>:/new/icons/Icons/textleft.png</normaloff>:/new/icons/Icons/textleft.png</iconset>
   </property>
   <property name="text">
    <string>Left</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
  </action>
  <action name="actionCenter">
   <property name="icon">
    <iconset resource="Note_Res.qrc">
     <normaloff>:/new/icons/Icons/textcenter.png</normaloff>:/new/icons/Icons/textcenter.png</iconset>
   </property>
   <property name="text">
    <string>Center</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionRight">
   <property name="icon">
    <iconset resource="Note_Res.qrc">
     <normaloff>:/new/icons/Icons/textright.png</normaloff>:/new/icons/Icons/textright.png</iconset>
   </property>
   <property name="text">
    <string>Right</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+D</string>
   </property>
  </action>
  <action name="actionJustify">
   <property name="icon">
    <iconset resource="Note_Res.qrc">
     <normaloff>:/new/icons/Icons/textjustify.png</normaloff>:/new/icons/Icons/textjustify.png</iconset>
   </property>
   <property name="text">
    <string>Justify</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+W</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="icon">
    <iconset resource="Note_Res.qrc">
     <normaloff>:/new/icons/Icons/textbold.png</normaloff>:/new/icons/Icons/textbold.png</iconset>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionItalics">
   <property name="icon">
    <iconset resource="Note_Res.qrc">
     <normaloff>:/new/icons/Icons/textitalic.png</normaloff>:/new/icons/Icons/textitalic.png</iconset>
   </property>
   <property name="text">
    <string>Italics</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionUnderline">
   <property name="icon">
    <iconset resource="Note_Res.qrc">
     <normaloff>:/new/icons/Icons/textunder.png</normaloff>:/new/icons/Icons/textunder.png</iconset>
   </property>
   <property name="text">
    <string>Underline</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionText_Size">
   <property name="text">
    <string>Text Size</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="text">
    <string>Font</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="Note_Res.qrc">
     <normaloff>:/new/icons/Icons/zoomin.png</normaloff>:/new/icons/Icons/zoomin.png</iconset>
   </property>
   <property name="text">
    <string>Zoom &amp;In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset resource="Note_Res.qrc">
     <normaloff>:/new/icons/Icons/zoomout.png</normaloff>:/new/icons/Icons/zoomout.png</iconset>
   </property>
   <property name="text">
    <string>Zoom &amp;Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="Note_Res.qrc">
     <normaloff>:/new/icons/Icons/fileopen.png</normaloff>:/new/icons/Icons/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="Note_Res.qrc">
     <normaloff>:/new/icons/Icons/filesave.png</normaloff>:/new/icons/Icons/filesave.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="Note_Res.qrc">
     <normaloff>:/new/icons/Icons/File Save As.png</normaloff>:/new/icons/Icons/File Save As.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp;As</string>
   </property>
   <property name="shortcut">
    <string>Meta+Alt+S</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="Note_Res.qrc">
     <normaloff>:/new/icons/Icons/exportpdf.png</normaloff>:/new/icons/Icons/exportpdf.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Export</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Meta+Alt+Q, Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Note_Res.qrc"/>
 </resources>
 <connections/>
</ui>
